Header header
string id 				# The id of the Markov chain (name of the map?)

MarkovArrayProba[] 	markov_chain	# array of the ids and proba for each region

